Source/Atomic/Resource/Image.cpp

```diff
@@ -1211,6 +1216,91 @@ bool Image::SaveJPG(const String& fileName, int quality) const
         return false;
 }
 
+bool Image::SaveDDS(const String& fileName) const
+{

```

Comment:
I am thinking saving DDS files should be in the domain of the tooling and not runtime.  Otherwise, SaveDDS should only be available on desktop and not increase iOS/Android/Emscripten runtimes